frontend/src/slices/auth/actions.ts

```diff
@@ -87,7 +91,9 @@ const signIn = createAsyncThunk<
 });
 
 const getCurrent = createAsyncThunk<
-  CustomerSignUpResponseDto | BusinessSignUpResponseDto,
+  | UserEntityObjectWithGroupT
+  | UserEntityObjectWithGroupAndBusinessT
+  | UserEntityObjectWithGroupAndDriverT,
```

Comment:
let's add some type for it?

Reply:
We have it. 

It is called UserSignInResponseDto, but i don't feel like to use it everywhere, this name is way too specific.

Reply:
We have it. 

It is called UserSignInResponseDto, but i don't feel like to use it everywhere, this name is way too specific.